extensions/gdx-setup/src/com/badlogic/gdx/setup/ExternalExtensionsDialog.java

```diff
@@ -0,0 +1,266 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+package com.badlogic.gdx.setup;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.swing.*;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.ItemEvent;
+import java.awt.event.ItemListener;
+import java.io.IOException;
+import java.util.*;
+
+
+import static java.awt.GridBagConstraints.*;
+
+/**
+ * Created by azakhary on 2/9/2015.
+ */
+public class ExternalExtensionsDialog extends JDialog {
```

Comment:
I added the vertical scroll that will appear in case it is too big. But I think when there will be around 5-10 extensions, it will look good if dialog height get's a little bit bigger, and then when even mroe extensions will be added, then they can enable scrolls. Or maybe when there will be that many extensions, it will make sense to make a more complicated dialog with more advanced options. But this is all for the future, for now there is just max size and scroll pane. which is ok I assume.